airbyte-webapp/src/packages/cloud/lib/domain/freeConnectorProgram/api.ts

```diff
@@ -0,0 +1,40 @@
+import { apiOverride } from "core/request/apiOverride";
+
+export type WebBackendGetFreeConnectorProgramInfoForWorkspaceResult = NonNullable<
+  Awaited<ReturnType<typeof webBackendGetFreeConnectorProgramInfoForWorkspace>>
+>;
+
+export interface WorkspaceIdRequestBody {
+  workspaceId: WorkspaceId;
+}
+
+/**
+ * Workspace Id from OSS Airbyte instance
+ */
```

Comment:
OSS Airbyte Instance? (I know this one is auto-generated, but that seems mislabelled, right?)

Reply:
Great catch!